libcxx/test/libcxx/containers/sequences/forwardlist/bool-conversion.pass.cpp

```diff
@@ -0,0 +1,28 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// <forward_list>
+
+#include <forward_list>
+struct Bool {
+  Bool() = default;
+  Bool(const Bool&) = delete;
+  operator bool() const { return true; }
+};
+
+struct Int {
+  Bool& operator==(Int) const {
+    static Bool b;
+    return b;
+  }
+};
+
+int main() {
+  std::forward_list<Int> l;
+  std::erase(l, Int{});
+}
```

Comment:
Some of the platforms we support require this main signature and and explicit return value.
```suggestion
int main(int, char**) {
  std::forward_list<Int> l;
  std::erase(l, Int{});
  
  return 0;
}
```